app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -0,0 +1,23 @@
+module V0
```

Comment:
Why would you need to bump versions for all the other APIs? You could have /v1/rx and /v2/education living alongside /v0/status